tests/consistent-package-json.test.ts

```diff
@@ -29,6 +29,7 @@ const KNOWN_TEMPLATE_KEYS = [
 ];
 
 const GLOB_PATH = './*';
+const CJS_EXPORT_KEYS = ['webpack', 'jest', 'jest-simple', 'babel'];
```

Comment:
This works, but claiming "export keys with these names are allowed to use the `default` export only" feels kinda weird. There's nothing inherently magical about these names that apply to all packages so blanket excluding them them feels odd.

I wonder If I'd feel better about this if the whitelist contained the package names too (e.g. `web-worker/babel`, so know we're talking about specific entrypoints in specific packages.

I'd also like to see a test that shows these keys use just a `default` value - currently we're omiting them from any validation